app/scripts/modules/amazon/src/function/configure/FunctionBasicInformation.tsx

```diff
@@ -0,0 +1,271 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import {
+  FormikFormField,
+  CheckboxInput,
+  AccountService,
+  IAccount,
+  IRegion,
+  IWizardPageComponent,
+  HelpField,
+  TextInput,
+  ReactSelectInput,
+  RegionSelectField,
+  Application,
+  ValidationMessage,
+} from '@spinnaker/core';
+
+import { FormikProps, FormikErrors, Field } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+import { Subject, Observable } from 'rxjs';
+
+const availableRuntimes = [
```

Comment:
Is there a way to fetch this information easily so we do not have to manually update every time this list changes? If not no worries, I am just not sure how often this can/will change.

Reply:
There is no API that serves this info currently.